README.md

```diff
@@ -46,6 +48,9 @@
    })
    ```
 
+  You can pass the secret key either directly in the runtime config or via a environment variable, or you can pass the path to a file that contains the secret key in the inline options. This is helpful when using docker secrets where docker passes the secret key via a file.
+  Remember to remove the secret key property from the runtime config when using the path as the property from the runtime config will be used then.
```

Comment:
The secret key is not part of the runtime configuration anymore, so this sentence can be removed I think.